04/styles/mobile.css

```diff
@@ -0,0 +1,75 @@
+.menu__navigation {
+    display: none;
+}    
+
+.menu > input:checked + .menu__navigation {
+    display: grid;
+    grid-template-areas:
```

Comment:
Używanie `grid` dla uważam za utrudniające życie ;) łatwiej by było zrobić to jako elementy blokowe - w moje opinii.
Oczywiście nie jest to błąd ;)